src/common/signing.props

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildType Condition="Exists('..\..\script\src\MetricsService.cs')">Internal</BuildType>
```

Comment:
One thing I've always wondered: why not test for `..\..\script\Key.snk` to detect an internal build as that's the thing we need for signing?